docs/plans/topology-target.yaml

```
@@ -12,6 +12,46 @@ files:
     target: polylogue/__main__.py
     owner: kernel
     reason: "kernel rule per #426"
+  - path: polylogue/api/__init__.py
+    loc: 76
+    target: polylogue/api/__init__.py
+    owner: stable
+    cross_cut: { api: async }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)